tests/server.py

```diff
@@ -296,7 +296,7 @@ def _get_mocked_response(self, method: str) -> Union[PayloadLike, bool]:
     async def _handle_body(self, body: bytes) -> None:
         try:
             await self._receive_payload(json.loads(body))
-        except IOError as ex:
+        except OSError as ex:
```

Comment:
Now I assume that this change was intentional,
but I do not know why it was made, can you clear that up for me? :)